snap/snapcraft.yaml

```diff
@@ -258,12 +258,21 @@ parts:
   qemu:
     override-pull: |
       versions_url=https://raw.githubusercontent.com/kata-containers/runtime/${SNAPCRAFT_PROJECT_VERSION}/versions.yaml
-      branch="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu-lite.branch)"
-      url="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu-lite.url)"
+      case "$(arch)" in
+        "aarch64")
+          branch="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu.architecture.aarch64.tag)"
+          url="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu.url)"
+        ;;
+
+        *)
+          branch="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu-lite.branch)"
+          url="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu-lite.url)"
+        ;;
+      esac
```

Comment:
@Pennyzct do me a favour and change the way qemu is downloaded, please use  `git` instead of `curl` that way           `keycodemapdb_commit` and `capstone_commit` won't be more needed

Reply:
Hi~ @devimc I have switched to `git`, ptal. ;)